tests/integration-tests/tests/cli_commands/test_cli_commands.py

```diff
@@ -310,13 +312,13 @@ def _test_pcluster_export_cluster_logs(s3_bucket_factory, cluster):
             {
                 "Action": "s3:GetBucketAcl",
                 "Effect": "Allow",
-                "Resource": f"arn:aws:s3:::{bucket_name}",
+                "Resource": f"arn:{partition}:s3:::{bucket_name}",
                 "Principal": {"Service": f"logs.{cluster.region}.amazonaws.com"},
             },
             {
                 "Action": "s3:PutObject",
                 "Effect": "Allow",
-                "Resource": f"arn:aws:s3:::{bucket_name}/*",
+                "Resource": f"arn:{partition}:s3:::{bucket_name}/*",
```

Comment:
What about retrieving the partition from the cluster rather than adding a new parameter?
It would be better to have something like:

```
cluster.partition
```

or

```
get_partition(cluster.region)
```

I think it would be a better approach because it's highly probable that whenever we need an ARN, we have the cluster or cluster region information already available in the scope (like in this specific case).

Reply:
Done